make_help_scripts/deploy_defines.py

```diff
@@ -0,0 +1,176 @@
+# Copyright (c) 2023 ros2_control maintainers
```

Comment:
@christophfroehlich in this are you checking the commits in other branches? Or only the master branch?

Reply:
what do you mean, sorry? :grimacing: 

the deploy_defines defines more settings and branch-distro maps for this repo as well as for all subrepos.

Reply:
Ok I thought this is also doing the PR stats, so I got confused.